treatment-service/instrumentation/prometheus.go

```diff
@@ -18,36 +18,60 @@ const (
 	FetchTreatmentRequestCount metrics.MetricName = "fetch_treatment_request_count"
 	// NoMatchingExperimentRequestCount is the key to measure no. of fetch treatment requests with no matching experiments
 	NoMatchingExperimentRequestCount metrics.MetricName = "no_matching_experiment_request_count"
+	// FetchTreatmentRequestDurationMsHelpString is the help string of the FetchTreatmentRequestDurationMs metric
+	FetchTreatmentRequestDurationMsHelpString string = "Histogram for the runtime (in milliseconds) of Fetch Treatment requests"
+	// ExperimentLookupDurationMsHelpString is the help string of the ExperimentLookupDurationMs metric
+	ExperimentLookupDurationMsHelpString string = "Histogram for the runtime (in milliseconds) of experiment lookup"
+	// FetchTreatmentRequestCountHelpString is the help string of the FetchTreatmentRequestCount metric
+	FetchTreatmentRequestCountHelpString string = "Counter for no. of Fetch Treatment requests with matching experiments"
+	// NoMatchingExperimentRequestCountHelpString is the help string of the NoMatchingExperimentRequestCount metric
+	NoMatchingExperimentRequestCountHelpString string = "Counter for no. of Fetch Treatment requests with no matching experiments"
 )
 
-// requestLatencyBuckets defines the buckets used in the custom Histogram metrics
-var requestLatencyBuckets = []float64{
+// RequestLatencyBuckets defines the buckets used in the custom Histogram metrics
+var RequestLatencyBuckets = []float64{
```

Comment:
Global definitions such as these are made public so that xp's plugin package can import them.